_sass/base.scss

```diff
@@ -0,0 +1,98 @@
+.site-header {
+	position: fixed;
+	top: 0;
+	left: 0;
+	width: 100%;
+	border-top: none;
+	border-bottom-color: transparent;
+	background-color: $dark-background-color;
+
+	min-height: 70px;
+	transition: all .5s;
+
+	display: flex;
+	align-items: center;
+
+	&.site-header-scrolled {
```

Comment:
lint: is the version without the ampersand the same as with?

Reply:
no, without means `.site-header-scrolled` is a child and with means it is both `.site-header` and `.site-header-scrolled`.

Reply:
oh I didn't see there was no space